packages/jest-environment-enzyme/README.md

```diff
@@ -17,11 +17,18 @@ But npm works too!
 $ npm install jest-environment-enzyme --save-dev
 ```
 
-### Setup
+With this library, you don't need to have your app install Enzyme, but you have to install one of Enzymes adapters.
+
+```
+yarn add enzyme-adapter-react-* --dev
```

Comment:
You missed moving the * here. `enzyme-adapter-*` 

Also, can you squash down to 1 commit?

Reply:
Those are two different fixes, explained by the commit messages, if I squash I have to get more evasive, like "fix docs" or something.